measurements/api/private.py

```diff
@@ -7,16 +7,21 @@
 """
 from datetime import datetime
 
+from six.moves.urllib.parse import urljoin
```

Comment:
IMHO, `six` is useless as `ooni-measurement` is py3-only

Reply:
Fair enough, but since we import and use it in other places already, I would rather do the removal of `six` globally than just here.